Node/angular-karma/package.json

```
@@ -0,0 +1,53 @@
+{
+  "name": "angular-proj1",
```

```suggestion
  "name": "angular-karma",
```
Or something a little bit more descriptive